web_widget_bokeh_chart/__manifest__.py

```diff
@@ -6,13 +6,13 @@
     "category": "Hidden",
     "summary": "This widget allows to display charts using Bokeh library.",
     "author": "ForgeFlow, " "Odoo Community Association (OCA)",
-    "version": "14.0.1.0.0",
+    "version": "14.0.2.3.1",
```

Comment:
I put this version as the installed version of bokeh is 2.3.1. WDYT?

Reply:
Yes, I saw it and I like it :wink: